File: AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/nativeads/carouselui/cards/InlineCarouselCardCallbacks.java
package com.applovin.enterprise.apps.demoapp.ads.applovin.nativeads.carouselui.cards;

/**
 * Represents callbacks for a view pager.
 * This is not part of the support library's stock pager (SdkViewPager),
 * You should set an OnPageChangeListener on the view pager and invoke these callbacks yourself.
 */
public interface InlineCarouselCardCallbacks
{
    /**
     * Called when a page becomes the foreground view.
     */
    public void onCardActivated();

    /**
     * Called when a page loses foreground focus.
     */
    public void onCardDeactivated();
}

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/nativeads/carouselui/cards/InlineCarouselCardCallbacks.kt
package com.applovin.apps.kotlindemoapp.nativeads.carouselui.cards

/**
 * Represents callbacks for a view pager.
 * This is not part of the support library's stock pager (SdkViewPager),
 * You should set an OnPageChangeListener on the view pager and invoke these callbacks yourself.
 */
interface InlineCarouselCardCallbacks
{
    /**
     * Called when a page becomes the foreground view.
     */
    fun onCardActivated()

    /**
     * Called when a page loses foreground focus.
     */
    fun onCardDeactivated()
}
